src/GitVersionCore/BuildServers/AppVeyor.cs

```diff
@@ -40,13 +35,9 @@ public override string[] GenerateSetParameterMessage(string name, string value)
         {
             var httpClient = GetHttpClient();
 
-            var body = new
-            {
-                name = $"GitVersion_{name}",
-                value = $"{value}"
-            };
+            var body = $"{{\"name\": \"GitVersion_{name}\",\"value\": \"{value}\"}}";
```

Comment:
I doubt we need to evaluate whether value is an Integer or not.

Reply:
Are you sure?

Reply:
Before it was going through JsonConvert.SerializeObject as object prior with the value of `value = $"{value}"` ergo it was turned into strings.

Reply:
Good point. Let's tackle that in a separate PR, if at all.